resources/schema/spidacalc/client/load_case.schema

```diff
@@ -251,6 +251,20 @@
       "description": "(Read-Only) all final strength factors and multipliers used during analysis (including default values)",
       "type": "object"
     },
+    "insulatorTypes": {
```

Comment:
let's name this something a little more specific, like "upliftInsulatorTypes" maybe